### PR TITLE
Contributors updates for Chapel 2.2

### PR DESCRIPTION
**Pull request by bradcray**
This is a refresh to the list of contributors to the Chapel 2.2 release relative to Chapel 2.1

### CODE REVIEW

**Comment by bradcray** [comment]
@insertinterestingnamehere : As a new contributor, would you double-check that I've spelled and alphabetized your name properly?  Also, if there's a link you'd like associated with your name (e.g., a professional page), let me know (completely optional, but an option if you like).

**Comment by insertinterestingnamehere** [comment]
Looks perfect. Thank you!

File: CONTRIBUTORS.md [hunk 1]
```diff
@@ -6,35 +6,27 @@ The following people have contributed to Chapel releases:
 Contributors to the Chapel 2.1 release
```
Comment (lydia-duncan) [inline CONTRIBUTORS.md:6]: Don't forget to update this to the current release number :)

Comment (bradcray) [reply CONTRIBUTORS.md]: When will I learn…?  🤦   Thanks Lydia!

Comment (lydia-duncan) [reply CONTRIBUTORS.md]: I don't think you forget *that* often XD

Comment (bradcray) [reply CONTRIBUTORS.md]: I think the git history would prove you wrong!  :D

File: CONTRIBUTORS.md [hunk 2]
```diff
@@ -3,38 +3,30 @@ Chapel Contributors
...
 * [Brad Chamberlain], [HPE]
 * Soohoon Choi, [HPE]
 * Sarah Coghlan, [HPE]
 * Jeremiah Corrado, [HPE]
-* Mark Dewing, individual contributor
 * Lydia Duncan, [HPE]
 * Daniel Fedorin, [HPE]
 * Michael Ferguson, [HPE]
-* Luca Ferranti, individual contributor
-* [Paul H. Hargrove], [Lawrence Berkeley National Laboratory]
```
Comment (bonachea) [inline CONTRIBUTORS.md:20]: Paul contributed PR #25787

File: CONTRIBUTORS.md [hunk 3]
```diff
@@ -3,38 +3,30 @@ Chapel Contributors
 
 The following people have contributed to Chapel releases:
 
-Contributors to the Chapel 2.1 release
+Contributors to the Chapel 2.2 release
 --------------------------------------
 * Jade Abraham, [HPE]
-* [Dan Bonachea], [Lawrence Berkeley National Laboratory]
```
Comment (bonachea) [inline CONTRIBUTORS.md:9]: Does my ongoing work with Spack/E4S packaging count as contribution?

**Review by bonachea — CHANGES_REQUESTED** [review]

**Comment by bradcray** [comment]
Thanks for noting both things, Dan!  I'm not sure how I missed Paul's PR (and I know there was another today), though GitHub's contributors page has failed me before; and we definitely count your E4S and Spack work as contributing to the release, it just didn't occur to me when I was taking my very GitHub-oriented pass on this file.

**Pull request merged**